app/assets/stylesheets/base/sidebar.scss

```diff
@@ -25,6 +25,7 @@
 
 #page-content-wrapper {
   width: 100%;
+  height: 100vh;
```

Comment:
Avoid having the select box cut when the field is at the last position in the form.